HomeWork/src/HomeWorkApp/HomeWorkApp.java

```diff
@@ -0,0 +1,46 @@
+package HomeWorkApp;
+
+public class HomeWorkApp {
+        public static void main(String[] args) {
+            printThreeWords();
+            checkSumSign();
+            printColor();
+            compareNumbers();
+        }
+
+        public static void printThreeWords()
+        {
+            System.out.print("Orange\nBanana\nApple\n");
+        }
+
+        public static void checkSumSign()
+        {
+            int a,b;
+            a=3; b=-5;
```

Comment:
нет особого смысла разделять объявление и инициализацию значением переменной
```suggestion
            int a=3; 
            int b=-5;
```